docs/src/api/class-tracing.md

```diff
@@ -0,0 +1,72 @@
+# class: Tracing
+
+Tracing object for collecting test traces that can be opened using
+Playwright CLI.
+
+## async method: Tracing.export
+
+Export trace into the file with the given name. Should be called after the
+tracing has stopped.
+
+### param: Tracing.export.path
+- `path` <[path]>
+
+File to save the trace into.
+
+## async method: Tracing.start
+
+Start tracing.
+
+```js
+await context.tracing.start({ name: 'trace', screenshots: true, snapshots: true });
+const page = await context.newPage();
+await page.goto('https://playwright.dev');
+await context.tracing.stop();
+await context.tracing.export('trace.zip');
+```
+
+```java
+context.tracing.start(page, new Tracing.StartOptions()
+  .setName("trace")
+  .setScreenshots(true)
+  .setSnapshots(true);
+Page page = context.newPage();
+page.goto('https://playwright.dev');
+context.tracing.stop();
+context.tracing.export(Paths.get("trace.zip")))
+```
+
+```python async
+await context.tracing.start(name="trace" screenshots=True snapshots=True)
+await page.goto("https://playwright.dev")
+await context.tracing.stop()
+await context.tracing.export("trace.zip")
+```
+
+```python sync
+context.tracing.start(name="trace" screenshots=True snapshots=True)
+page.goto("https://playwright.dev")
+context.tracing.stop()
+context.tracing.export("trace.zip")
+```
+
+### option: Tracing.start.name
+- `name` <[string]>
+
+If specified, the trace is going to be saved into the file with the
```

Comment:
In what directory will the file be saved? I think we are missing docs for `traceDir` option?